packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -17,32 +18,43 @@ export function useFrequentlyBoughtTogether<TObject>({
   recommendClient,
   threshold,
   transformItems: userTransformItems,
-}: GetFrequentlyBoughtTogetherProps<TObject>) {
-  const [result, setResult] = useState<GetRecommendationsResult<TObject>>({
-    recommendations: [],
-  });
+  initialState,
+}: GetFrequentlyBoughtTogetherProps<TObject> & {
+  initialState?: InitialResults<TObject>;
+}) {
+  const initialResults = initialState ?? { recommendations: [] };
+  const [result, setResult] = useState<GetRecommendationsResult<TObject>>(
+    initialResults
+  );
+  const renderRef = useRef(false);
```

Comment:
We could make this ref more explicit with something like this:

```suggestion
  const isFirstRenderRef = useRef(true);
```